L1Trigger/TrackFindingTracklet/interface/DR.h

```diff
@@ -0,0 +1,80 @@
+#ifndef L1Trigger_TrackFindingTracklet_DR_h
+#define L1Trigger_TrackFindingTracklet_DR_h
+
+#include "L1Trigger/TrackTrigger/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+#include "L1Trigger/TrackFindingTracklet/interface/ChannelAssignment.h"
+
+#include <vector>
+
+namespace trklet {
+
+  /*! \class  trklet::DR
+   *  \brief  Class to emulate Duplicate Removal f/w
```

Comment:
Change comment to "Bit-accurate emulation of duplicate removal".

Reply:
done